aiphysim/losses.py

```diff
@@ -178,13 +178,25 @@ def set_args(self, batch, predictions, model):
 class SpaceTimeLoss(BaseLoss):
     def set_args(self, batch, predictions, model):
 
-        self.args = []
+        encoded_ts, decoded_ts, next_zs, next_decoded_ts = predictions
+
+        time_series = batch["data"]
+
+        self.args = [
+            ("l2", (model,)),
```

Comment:
The l2 is just for regularization, I think you still need the reconstruction loss

Reply:
This is [WIP].